From c33f43c87bca61ff0ea9312a2d6ef6a0def32026 Mon Sep 17 00:00:00 2001
From: Felix Palmen <felix@palmen-it.de>
Date: Mon, 25 Nov 2013 01:02:23 +0100
Subject: Follow GNU coding standards for prefix and datadir


diff --git a/src/Makefile b/src/Makefile
index fad6f55..15a74b4 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -13,19 +13,20 @@ PROGS = ar65  \
         sp65
 
 ifdef prefix
-  CA65_INC = $(prefix)/share/cc65/asminc
-  CC65_INC = $(prefix)/share/cc65/include
-  LD65_LIB = $(prefix)/share/cc65/lib
-  LD65_OBJ = $(prefix)/share/cc65/lib
-  LD65_CFG = $(prefix)/share/cc65/cfg
+  datarootdir = $(prefix)/share
+  datadir = $(datarootdir)/cc65
 else
-  CA65_INC := $(abspath ../asminc)
-  CC65_INC := $(abspath ../include)
-  LD65_LIB := $(abspath ../lib)
-  LD65_OBJ := $(abspath ../lib)
-  LD65_CFG := $(abspath ../cfg)
+  prefix = $(abspath ..)
+  datarootdir = $(prefix)
+  datadir = $(datarootdir)
 endif
 
+CA65_INC = $(datadir)/asminc
+CC65_INC = $(datadir)/include
+LD65_LIB = $(datadir)/lib
+LD65_OBJ = $(datadir)/lib
+LD65_CFG = $(datadir)/cfg
+
 CFLAGS += -MMD -MP -O -std=c89 -I common \
           -Wall -Wextra -Wno-char-subscripts -Werror \
           -DCA65_INC=$(CA65_INC) -DCC65_INC=$(CC65_INC) \
-- 
1.7.10.4

